codegen/masm2/src/lib.rs

```diff
@@ -34,6 +36,10 @@ pub use self::{
 pub fn register_dialect_hooks(context: &midenc_hir2::Context) {
     use midenc_dialect_hir as hir;
 
+    context.register_dialect_hook::<builtin::BuiltinDialect, _>(|info, _context| {
```

Comment:
You'll want to add these ops to the `eval` crate's `register_dialect_hooks` function as well, since the `Eval` trait is registered as a operation trait.

Reply:
This file was deleted in your branch, so don't see after rebase. Since the tests are not failing I assume you also added them in `codegen/masm/src/lib.rs`, so we should be fine.

Reply:
Yeah the `masm2` crate was moved to `masm` (replacing the old `masm` contents)